### DISCUSSION TITLE
не видит плоты

### DISCUSSION THREAD

**Discussion by ArtemKasper** [Chia-Network/chia-blockchain#6725] [Support]
после перезагрузки компьютера, программа не видит плоты засеянные на двух засеянных дисках, видит плоты только на последнем диске, который засеял 3 плота. помогите.

**Comment by loppefaaret** [comment]
COMPLETE CONFIG RESET

Close down the Chia app, and open up a PowerShell window, and execute the below commands one at a time:
```
mv ~\.chia\mainnet ~\.chia\old
cd ~\AppData\Local\chia-blockchain\app-1.1.7\resources\app.asar.unpacked\daemon\
.\chia.exe init
mv ~\.chia\old\db ~\.chia\mainnet\
mv ~\.chia\old\wallet ~\.chia\mainnet\
```

**Reply by ArtemKasper** [reply]
к сожалению теперь плоты вообще перестало видить

**Reply by loppefaaret** [reply]
you have to add them back in now, after all settings have been reset

**Reply by ArtemKasper** [reply]
спасибо, помогло,

**Reply by ArtemKasper** [reply]
единственная проблема теперь, долгая синхронизация

**Comment by BrandtH22** [comment]
@ArtemKasper , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
